recipes/fontconfig/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+
+if [[ `uname` == 'Darwin' ]];
+then
+    export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+elif [[ `uname` == 'Linux' ]];
+then
+    export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
+fi
```

Comment:
So, it turns out if something tries to load the libraries we have built before fixing them with `install_name_tool` or `patchelf`, these may not work properly in some cases. While I am still a bit fuzzy on details, I believe this has to do with some dependency libraries using rpaths. By doing this, we can work around this issue during testing. It ends up being needed for install in this case as some binaries are run during the install process that run into issue otherwise. Fortunately, this has no effect on the package. After running `install_name_tool` or `patchelf` on the libraries and binaries following the build step, we find this is no longer necessary in the final package.